R/add_smr_type.R

```diff
@@ -103,7 +103,7 @@ add_smr_type <- function(recid,
 
   # Situation where an Acute/GLS recid is given but no ipdc marker
   if (any(recid %in% c("01B", "GLS")) & missing(ipdc)) {
-    cli::cli_abort(
+    cli::cli_warn(
       "An {.var ipdc} vector has not been supplied, and therefore Acute/GLS
                    records cannot be given an {.var smrtype}"
     )
```

Comment:
I think this should still be an error. If `ipdc` isn't supplied that's incorrect and should error which is different from the variable being supplied and some being `NA`.